### ISSUE TITLE
Make your readme better god damnit

### ISSUE THREAD

**Issue by Samuel-Clarke123** [torinberger/VEG#3]
lotsa love and all but i cant understand what is going on without a readme, noone can.

also, 49 repos? damn someone loves git(hub?).
I only ever got up to 17 and half are trash repos.
gw man

**Comment by torinberger** [comment]
Yes the readme is shit, I just cant be bothered. Good to see you man. As for what this is? I cant explain it to myself but in a nutshell, basically a virtual economy game that runs in the CLI and is built in nodeJS. I know, I know, its very strange. I just love nodeJS okay. Hope work is going well

**Comment by Samuel-Clarke123** [comment]
Honestly, node is amazing, but don't be afraid to use other applications.
It's good even if its annoying, you never know what library employers will
want, and not all of them have the same methodology.
To explain what I mean, try learning react / redux. Good luck lmao.

**Comment by torinberger** [comment]
Lol, the idea was whether I could do it in nodeJS. But nvm, thanks for the advice
